app/controllers/account_controller.rb

```diff
@@ -1,11 +1,11 @@
-class ProfileController < ApplicationController
+class AccountController < ApplicationController
   before_action :confirm_two_factor_authenticated
   layout 'card_wide'
 
   def index
```

Comment:
I just remebered Rails' support for singular resources: WDYT of `AccountController#show`: 

http://guides.rubyonrails.org/routing.html#singular-resources